### ISSUE TITLE
Modbus + DSMR P1

### ISSUE THREAD

**Issue by skoef** [ThaStealth/alfen_modbus#5]
Hi, this is not so much of an issue as a question: do you know if it is possible to use both the DSMR (P1) protocol and access modbus registers at the same time? I'm using Solar Charging on my Alfen Eve Single S-line with a serial cable to the energy meter in my house. I would also like to read out metrics with something like your homeassistant integration, but TCP port 502 (the port for modbus as I understood) is unreachable at the moment on the IP of the charger.

Is there any middle ground here so I can use both at the same time?

**Comment by ThaStealth** [comment]
I don't believe it is possible, since the charger only wants to have one source of truth to look at (either the P1 or the modbus)

**Comment by w4iq3** [comment]
Actually you can do this now, using ACE installer there is a setting in the load balancing menu: P1 + EMS monitoring. I have a wired P1 connection to my grid meter, and use this integration to read out via modbus.
